### PR TITLE
Update to `context.tf`. Add `primary` and `secondary` cluster resources

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Update to `context.tf`
* Add `primary` and `secondary` `resource "aws_rds_cluster"`

## why
* Standardization and interoperability
* Keep the module up to date
* If Cluster is part of a Global Cluster, use the `lifecycle` configuration block `ignore_changes` argument to prevent Terraform from showing differences for `replication_source_identifier` argument instead of configuring this value (and we can't use `dynamic` since it's not supported in `lifecycle` blocks).

```
A dynamic block can only generate arguments that belong to the resource type, data source, provider 
or provisioner being configured. It is not possible to generate meta-argument blocks such as 
lifecycle and provisioner blocks, since Terraform must process these before 
it is safe to evaluate expressions.
```

## references
* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster#replication_source_identifier
* https://www.terraform.io/docs/configuration/expressions.html#dynamic-blocks

## related
* Closes #78

### CODE REVIEW

**Review by htplbc — APPROVED** [review]

File: outputs.tf [hunk 1]
```diff
@@ -4,27 +4,27 @@ output "database_name" {
 }
 
 output "master_username" {
-  value       = join("", aws_rds_cluster.default.*.master_username)
+  value       = coalesce(join("", aws_rds_cluster.primary.*.master_username), join("", aws_rds_cluster.secondary.*.master_username))
```
Comment (Nuru) [inline outputs.tf:7]: I expect this will fail when the module is disabled. `coalesce` throws an error if all its arguments are empty.

**Review by Nuru — CHANGES_REQUESTED** [review]

File: outputs.tf (hunk 1 above)
Comment (drselump14) [reply outputs.tf:7]: I face the same issue here

Comment (aknysh) [reply outputs.tf:7]: this was fixed in https://github.com/cloudposse/terraform-aws-rds-cluster/pull/80 and is already in `master`

**Pull request merged**